index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div id="background-mobile">
+      <div id="status-bar"></div>
+      
+      <header>
+        <a href="#">
+          <h2>Coder</h2>  
+        </a>
```

Comment:
- [ ] [OPTIONAL] -Great job wrapping your logo in an anchor tag. To make it look like in the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and remove the underline, putting `text-decoration: none;` will fix this for you.